examples/quick-start-guide.md

```diff
@@ -0,0 +1,109 @@
+# Quick Start Guide
```

Comment:
I think it is better to use the latest M2 release of Ballerina for this tutorial rather than the 0.991.
You can download the latest release from [1] and vs code vsix plugin from [2]

[1] https://product-dist.ballerina.io/dev/0.992.0/jballerina-tools-0.992.0-m2-SNAPSHOT.zip
[2] https://product-dist.ballerina.io/dev/0.992.0/ballerina-0.992.0.vsix